Gemfile

```diff
@@ -192,24 +192,13 @@ group :test, :cucumber do
   gem 'simplecov', require: false
   gem 'timecop', require: false
 
-  # Simplifies shared transactions between server and test threads
-  # See: http://technotes.iangreenleaf.com/posts/the-one-true-guide-to-database-transactions-with-capybara.html
-  # Essentially does two things:
-  # - Patches rails to share a database connection between threads while Testing
-  # - Pathes rspec to ensure capybara has done its stuff before killing the connection
-  # Causing problems in Rails 6. Remove from Rspec, left in place for cucumber, but can
-  # probably be remove there as well.
-  gem 'transactional_capybara'
-
   # Keep webdriver in sync with chrome to prevent frustrating CI failures
   gem 'webdrivers'
 end
 
 group :cucumber do
   gem 'cucumber_github_formatter'
   gem 'cucumber-rails', require: false
-  gem 'mime-types'
-  gem 'rubyzip'
```

Comment:
These are both transient dependencies of other gems that we don't use directly. They were added historically when either we needed to pin the version, or the dependencies were incorrectly declared. Letting bundler resolve this now.